ts/src/flexible-event/main.ts

```diff
@@ -114,6 +114,14 @@ if (options.json_file !== undefined) {
   if (options.windows.value.length !== options.buckets.value.length) {
     throw new Error('windows and buckets must have same length')
   }
+  if (
+    (options.attribution_scope_limit === undefined) !==
+    (options.max_event_states === undefined)
+  ) {
+    throw new Error(
+      'attribution_scope_limit and max_event_states must be set / unset at the same time'
```

Comment:
max_event_states could be optional when attribution_scooe_limit is set and default to 3? But it’s probably fine to require it for this utility.

Reply:
ACK. lmk if you think more changes are needed here.